tapir/coop/templates/coop/membership_resignation_list.html

```diff
@@ -19,43 +19,52 @@
+                        data-bs-toggle="dropdown">{% translate "Export" %}</button>
+                <div class="dropdown-menu dropdown-menu-right">
+                    {% for format in view.export_formats %}
+                        <a class="dropdown-item" href="{% export_url format %}">
+                            download <code>.{{ format }}</code>
+                        </a>
+                    {% endfor %}
                 </div>
                 <a class="{% tapir_button_link_to_action %}"
                    href="{% url 'coop:resign_new_membership' %}">
                     <span class="material-icons">add_circle_outline</span>
                     {% translate "Resign new member" %}
                 </a>
+                <button class="{% tapir_button_link_to_action %}"
+                        type="button"
+                        data-bs-toggle="collapse"
+                        data-bs-target="#collapseFilter"
+                        aria-expanded="false"
+                        aria-controls="collapseFilter">
+                    {% translate "Filters" %}
+                    <span class="material-icons">unfold_more</span>
+                </button>
             </div>
         </div>
         <br>
-        <form method="get">
-            <div class="px-3 mt-1">
-                <div class="member-filter-form-filters">{% bootstrap_form filter.form %}</div>
-                <div class="member-filter-form-buttons">
-                    <button class="{% tapir_button_link %}">
-                        <span class="material-icons">filter_alt</span>{% translate 'Filter' %}
-                    </button>
-                    <a class="{% tapir_button_link %}"
-                       href="{% url 'coop:resigned_members_list' %}">
-                        <span class="material-icons">clear</span>
-                        {% translate "Clear all filters" %}
-                    </a>
+        <div class="collapse" id="collapseFilter">
+            <form method="get">
+                <div class="px-3 mt-1">
+                    <div class="member-filter-form-filters">{% bootstrap_form filter.form %}</div>
+                    <div class="member-filter-form-buttons">
+                        <button class="{% tapir_button_link %}">
+                            <span class="material-icons">filter_alt</span>{% translate 'Filter' %}
+                        </button>
+                        <a class="{% tapir_button_link %}"
+                           href="{% url 'coop:resigned_members_list' %}">
+                            <span class="material-icons">clear</span>
+                            {% translate "Clear all filters" %}
+                        </a>
+                    </div>
                 </div>
-            </div>
-        </form>
-        <ul>
+            </form>
+        </div>
+        <ul class="table-responsive" style="padding-left: 0px">
```

Comment:
Same as `draftuser_list.html`: lets set the padding for top-left-right, set  the appropriate classes on the `ul` element and put `table-responsive` on a new dedicated element.